.github/workflows/get-then-push-data.yml

```diff
@@ -0,0 +1,33 @@
+# Automated pull of data needed for the public directory from an HQ UCR API endpoint.
+# Data is pushed to S3 bucket for use by the static site. Authentication with HQ
+# occurs via an API key, while authentication with AWS happens via OIDC.
+name: get-then-push-data
+on:
+  schedule:
+  # Run every 3 hours at 1:30AM, 4:30 AM, 7:30 AM... every day
+    - cron: '30 1,4,7,10,13,16,19,22 * * *'
+env:
+  BUCKET_NAME : ${{ secrets.S3_BUCKET_NAME }}
+  AWS_REGION : ${{ secrets.AWS_REGION }}
+  HQ_API_KEY : ${{ secrets.ADDISONS_TEST_HQ_API_KEY }}
+jobs:
+  get-data-then-push:
+    runs-on: ubuntu-latest
+    # Get JWT token from GitHub's OIDC provider
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Get data from the HQ API
+        run: |
+          curl -H "Authorization: ApiKey ${{ secrets.ADDISONS_TEST_API_USERNAME }}:${{ secrets.ADDISONS_TEST_HQ_API_KEY }}" \
+          'https://www.commcarehq.org/a/${{ secrets.PROJECT_NAME }}/api/v0.5/configurablereportdata/${{ secrets.REPORT_ID }}/' >> directory_data.json
```

Comment:
I imagine in the future we'll want to do some transformation of this returned value.  At that point can we basically just replace this with a call to a python or JS script?

Reply:
Yup looks like we can run scripts we store in the repo:

https://docs.github.com/en/actions/learn-github-actions/essential-features-of-github-actions#adding-scripts-to-your-workflow